test/JDBC/input/sp_describe_first_result_set-vu-verify.mix

```diff
@@ -0,0 +1,4 @@
+-- tsql
```

Comment:
Is there any reason for using .mix extension?

Reply:
Changed .mix files to .sql